### PR TITLE
[Iteration 6] v4.0-beta

### PR DESCRIPTION
**Pull request by Jeckerson**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "require": {
         "php": ">=7.2",
         "ext-openssl": "*",
+        "ext-apc": "*",
```
Comment (sergeyklay) [inline composer.json]: can we move this to `suggest`  sections ?

Comment (Jeckerson) [reply composer.json]: 8e8fb77

Comment (sergeyklay) [reply composer.json]: @Jeckerson

In fact there is a script to install apc(u) on Travis CI: https://github.com/phalcon/cphalcon/blob/b2c72e83c5ed4202075566ba2642c28c6672e32c/.ci/install-php-extensions.sh#L14-L28

**Comment by sergeyklay** [comment]
@Jeckerson Any idea what happens with Sqlite?

**Comment by Jeckerson** [comment]
@sergeyklay Yes, Phinx migrations add prefix `.sqlite3` during migrations which cause problem to connect to the DB inside DB Provider.

Example:
Inside .env file `DB_NAME=/tmp/test.sqlite`
Which, after migrations, will generate file `/tmp/test.sqlite.sqlite3`

**Pull request merged**